AutolayoutPractice/RemindersList/Views/ReminderCell.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  ReminderCell.swift
+//  AutolayoutPractice
+//
+//  Created by Amanda Benavente on 6/17/24.
+//
+
+import Foundation
+import UIKit
+
+class ReminderCell: UITableViewCell {
+    static let identifier = "reminderCell"
+    
+    var reminder: Reminder? {
+        didSet {
+            title.text = reminder?.title
```

Comment:
1. `func setup(title: String, summary: String, dueDateString: String)` - called in `ReminderViewController`
2. Configuration

```
struct ReminderCellConfiguration {
    let title: String
    let description: String
    let dueDateString: String
}

class ReminderCell {
    func setup(configuration: ReminderCellConfiguration) {
          title.text = configuration.title
          ...
    }
```